packages/camera/camera_android_camerax/android/src/test/java/io/flutter/plugins/camerax/CameraPermissionsManagerTest.java

```diff
@@ -0,0 +1,89 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.camerax;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+import android.content.pm.PackageManager;
+import io.flutter.plugins.camerax.CameraPermissionsManager.CameraRequestPermissionsListener;
+import io.flutter.plugins.camerax.CameraPermissionsManager.ResultCallback;
+import org.junit.Test;
+
+public class CameraPermissionsManagerTest {
```

Comment:
This is the same as [CameraPermissions.java](https://github.com/flutter/plugins/blob/main/packages/camera/camera_android/android/src/test/java/io/flutter/plugins/camera/CameraPermissionsTest.java) in the old plugin/